buildconfig is meant to be executed on the host, so it has to be compiled
using $(HOSTCC), not $(CC).

Signed-off-by: Luca Ceresoli <luca@lucaceresoli.net>
---
 OS/Makefile-Base |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/OS/Makefile-Base b/OS/Makefile-Base
index 29a6ad3..420ba60 100644
--- a/OS/Makefile-Base
+++ b/OS/Makefile-Base
@@ -114,8 +114,8 @@ allexim: config.h $(EXIM_MONITOR) exicyclog exinext exiwhat \
 
 # Targets for special-purpose configuration header builders
 buildconfig: buildconfig.c
-	@echo "$(CC) buildconfig.c"
-	$(FE)$(CC) $(CFLAGS) $(INCLUDE) -o buildconfig buildconfig.c $(LIBS)
+	@echo "$(HOSTCC) buildconfig.c"
+	$(FE)$(HOSTCC) $(HOSTCFLAGS) $(INCLUDE) -o buildconfig buildconfig.c
 
 
 # Target for the exicyclog utility script
